mypy.ini

```diff
@@ -0,0 +1,1070 @@
+[mypy]
+python_version = 3.9
+pretty = false
+exclude = (?x)(
+          docs|
+          examples
+          )
+
+enable_error_code = ignore-without-code
+
+# We can get to passing mypy's strict mode by incrementally changing the
+# following global settings to true. This comes from mypy's documentation.
+
+warn_unused_configs = true
+warn_redundant_casts = true
+warn_unused_ignores = true
+strict_equality = true
+check_untyped_defs = false
+disallow_subclassing_any = false
+disallow_untyped_decorators = false
+disallow_any_generics = false
+disallow_untyped_calls = false
+disallow_incomplete_defs = false
+disallow_untyped_defs = false
+no_implicit_reexport = false
+warn_return_any = false
+extra_checks = false
+
+# The following sections specify configuration settings specific to
+# different subpackages and modules.
+
+# The disable_error_code configuration variable specifies that certain
+# typing errors should not be reported for specific modules. As we
+# gradually make Astropy consistent with mypy's strict mode, we can
+# remove the disabled error codes file-by-file.
+
```

Comment:
Almost everything below here was created automatically.

Reply:
It would be great to add separation comments between the modules

e.g.


```python
[mypy-astropy.conftest]
disable_error_code = arg-type,import-not-found

#####################################################################

[mypy-astropy.constants.codata2010]
disable_error_code = var-annotated
```